docs/versioned_docs/version-2.11.0/concepts/architecture.md

```diff
@@ -0,0 +1,165 @@
+
+
+### CSI Driver (Dynamic Volume Provisioner) 
+
+CSI Drivers act as the facilitators for managing the life-cycle of volumes within Kubernetes. The CSI Driver operations are controlled or customized by the parameters specified in `StorageClass`. The CSI drivers comprise of three layers:
+- Kubernetes or Orchestrator functionality - that is native to Kubernetes and bind the application to the volumes 
+- Kubernetes CSI layer - that translates the Kubernetes native calls into CSI calls - passing the information provided by users in a standard way to the CSI Drivers 
+- Storage Drivers - which are CSI complaint and work very closely with the Kubernetes CSI layer to receive the requests and process them. 
+
+The Storage Drivers are responsible for:
+- Exposing the capabilities of the Data engines
+- Either directly interacting with the Data Engine or the Data Engine Operators to perform volume creation and deletion operations
+- Interface with the Data engines to attach/detach the volumes to the nodes where containers consuming the volumes are running
+- Interface with standard linux utilities to format, mount/unmount the volumes to the containers
+
+
+### Plugins
+
+OpenEBS focuses on storage operations and provides plugins for other popular tools for performing the operations that fall outside of the core storage functionality but are very important for running OpenEBS in production. Examples of such operations are:
+- Application Consistent Backup and Recovery (provided via integrations into Velero) 
+- Monitoring and Alerting ( provided via integrations into Prometheus, Grafana, Alert manager) 
+- Enforcing Security Policies ( provided via integrations with PodSecurityPolicies or Kyerno) 
+- Logging ( provide via integration to any standard Logging stack setup by administrators like ELK, Loki, Logstash) 
+- Visualizations  (provided via standard Kubernetes Dashboards or custom Grafana dashboards)
+
+
+### CLI 
+
+All the management functions on OpenEBS can be carried out via `kubectl` as OpenEBS uses Custom Resources for managing all of its configurations and reporting the status of the components. 
+
+In addition, OpenEBS also has released as alpha version `kubectl plugin` to help with providing information about the pools and volumes using a single command that aggregates the information obtained via multiple `kubectl` commands.
+
+
+ 
+## See Also:
+
+[Understanding Data Engines](/docs/next/casengines.html) [Understanding Mayastor](https://mayastor.gitbook.io/introduction/) [Understanding Local PV](/docs/next/localpv.html) [Understanding cStor](/docs/next/cstor.html) [Understanding Jiva](/docs/next/jiva.html)
```

Comment:
Is this properly formatted ? Please review it once with the actual output.

Reply:
yes, the see also section should be in a single line

Reply:
This section is coming as expected 
![Screenshot from 2021-08-04 12-05-33](https://user-images.githubusercontent.com/10995416/128133359-4177667b-b006-4f65-9e7f-da1e95730b13.png)